metadata/metadata_service/proxy/atlas_proxy.py

```diff
@@ -1456,3 +1456,26 @@ def get_lineage(self, *,
 
     def get_feature(self, *, feature_uri: str) -> Feature:
         pass
+
+    def get_resource_description(self, *,
```

Comment:
I wonder what others think of the generality here. It raises the obvious future question of whether to do `get_resource_description` with Table as `resource_type` or use `get_table_description`.

Keeping with the per-entity boilerplate wagon seems more straightforward, perhaps.

Reply:
One of the benefits we get from using the generic `get_resource_description` is that we can pretty much reuse all of the same code for any resource types with the one caveat being that they must all follow a similar data model within the DB. For instance, tables, dashboards, features, etc. must all have a "Description" and use the same relationship label. This way the # of one-off functions to support N number of resource types stays at 1 per action (e.g. get description, delete desc, etc.)

Reply:
I don't see why we would ever change this relationship for future entities so I think we should work towards making these methods general wherever we can.

Reply:
+1 to have these functions generic enough to support all kinds of entities/resources, and the implementation can be different based on the `resource_type`